src/pages/about/index.tsx

```diff
@@ -138,6 +139,22 @@ const Funding = () => {
   )
 }
 
+const Attributions = () => {
+  return (
+    <Widget>
+      <h2>Attributions</h2>
+      Bus ifmage by{' '}
+      <a
+        href="https://www.freepik.com/free-vector/passengers-waiting-bus-city-queue-town-road-flat-vector-illustration-public-transport-urban-lifestyle_10173277.htm#query=public%20transportation&position=0&from_view=search&track=ais&uuid=70a79b38-20cb-42b8-9dde-b96a68088522"
+        target="_blank"
+        rel="noopener noreferrer">
```

Comment:
shouldn't we use `rel="noopener noreferrer nofollow"`?

Reply:
good idea.